packages/nodejs/src/ServiceClient.js

```diff
@@ -94,12 +104,12 @@ class ServiceClient extends BaseServiceClient {
   }
 
   _getGrpcChannelCredentials(serviceEndpoint) {
-    if(serviceEndpoint.protocol === 'https:') {
+    if (serviceEndpoint.protocol === 'https:') {
```

Comment:
lets stndrdize the indentaion too , ths should not come up in diff

Reply:
Resolved